src/components/AddItemForm.js

```diff
@@ -26,49 +26,61 @@ const AddItemForm = () => {
     setNewItem('');
   };
   return (
-    <form onSubmit={handleSubmit}>
-      <label htmlFor="newItem">Item name:</label>
-      <input
-        className="input"
-        type="text"
-        id="newItem"
-        value={newItem}
-        onChange={handleItemAdd}
-        name="newItem"
-        placeholder="Add Item"
-        required
-        ref={newItemInputRef}
-      />
-
-      <fieldset>
-        <legend>How soon will you buy this again ?</legend>
-        <input
-          type="radio"
-          id="soon"
-          value="7"
-          checked={timeframe === '7'}
-          onChange={handleSelect}
-          name="timeframe"
-        />
-        <label htmlFor="soon">Soon</label>
+    <form onSubmit={handleSubmit} className="flex flex-col items-centre">
+      <label htmlFor="newItem" className="flex justify-centre">
         <input
-          type="radio"
-          id="kind-of-soon"
-          value="14"
-          checked={timeframe === '14'}
-          onChange={handleSelect}
-          name="timeframe"
+          className="input"
+          type="text"
+          id="newItem"
+          value={newItem}
+          onChange={handleItemAdd}
+          name="newItem"
+          placeholder="Enter an item..."
+          required
+          ref={newItemInputRef}
         />
-        <label htmlFor="kind-of-soon">kind of soon</label>
-        <input
-          type="radio"
-          id="not-soon"
-          value="30"
-          checked={timeframe === '30'}
-          onChange={handleSelect}
-          name="timeframe"
-        />
-        <label htmlFor="not-soon">not-Soon</label>
+      </label>
+
+      <fieldset className="flex flex-col gap-y-1 mt-4">
+        <legend className="text-xl">
+          How soon will you buy this item again ?
+        </legend>
+        <label htmlFor="soon">
+          <input
+            className="border-2 border-cyan-500 form-radio text-cyan-500 h-5 w-5 mb-5"
```

Comment:
Maybe we could use `@apply` here for the repetative classNames for each radio button. I'd also look at the colours so it matches the input field - either both blue or both cyan?

Reply:
I agree. Using @apply is appropriate here.

Reply:
IMO cyan looks better.

Reply:
I agree, it goes well with the other colours.

Reply:
good idea! I'll use it and also change the input tobe cyan as well